### DISCUSSION TITLE
add iconsax to lib

### DISCUSSION THREAD

**Discussion by MM25Zamanian** [lit/lit#2398] [Ideas]
Hello
What do you think about an icon component?
The icon pack I have in mind **(if you agree)** is [**iconsax**](https://iconsax.io/)

The preview I have in mind is as follows:
```html
<lit-iconsax size="24" color="fff"></lit-iconsax>
```

**Comment by kevinpschaaf** [comment]
This sounds like a great web component to built with Lit (or even vanilla Web Components, given it's likely so simple). We don't include individual components in the core libraries, but would encourage you to explore [publishing](https://lit.dev/docs/tools/publishing/) one yourself.

**Reply by MM25Zamanian** [reply]
Hello. 
Thank you for encouraging me.
But I want to work with professionals.
If you help me in making this package. 
I'm really happy

Help me??

**Reply by thescientist13** [reply]
I think if you're looking to get started and are unsure how, the Lit (and friends) Slack community is a great first step.  You can find the link and other support resources at the very bottom of the Lit home page.
https://lit.dev/
